concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md

```diff
@@ -6,6 +6,8 @@ The DataGrid uses the DevExtreme [Form](/api-reference/10%20UI%20Components/dxFo
 
 For example, you can specify the [items](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/items.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items') array to reorder editable fields (or "simple items", as they are called in the Form UI component), or organize them in groups and tabs.
```

Comment:
If a group is preferable, move it to the first position: group, tabs, or leave items ungrouped.